src/logger/logger.ts

```diff
@@ -0,0 +1,670 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as os from 'os';
+import * as path from 'path';
+
+import { Writable } from 'stream';
+import { Logger as PinoLogger, pino } from 'pino';
+import { Env } from '@salesforce/kit';
+import { isKeyOf, isString } from '@salesforce/ts-types';
+import { Global, Mode } from '../global';
+import { SfError } from '../sfError';
+import { unwrapArray } from '../util/unwrapArray';
+import { MemoryLogger } from './memoryLogger';
+import { cleanup } from './cleanup';
+
+/**
```

Comment:
most of the bunyan-specific stuff here is deprecated.  Most become no-ops.

Reply:
Now removed since we decided to bump major